cmd/mattermost/commands/config.go

```diff
@@ -308,6 +308,9 @@ func UpdateMap(configMap map[string]interface{}, configSettings []string, newVal
 		if len(configSettings) == 1 {
 			return errors.New("unable to set multiple settings at once")
 		}
+		if value.Len() == 0 {
+			return fmt.Errorf("unable to find a setting with that name %s", configSettings[1])
```

Comment:
Language here might be hard to parse, perhaps:
```suggestion
			return fmt.Errorf("unable to find a setting named: %s", configSettings[1])
```